config/config.go

```diff
@@ -28,6 +28,7 @@ const (
 	FieldNameTag   = "yaml"
 	ReadonlyTag    = "readonly"
 	DebugFlag      = "debug"
+	TelemetryFlag  = "no-telemetry"
```

Comment:
Instead, use the `TELEMETRY_DISABLED` environment variable mentioned in https://github.com/kubeshark/hub/pull/4#discussion_r1305936600

Reply:
I think it wasn't used at all here, removed.